src/diffusers/models/attention.py

```diff
@@ -332,9 +329,9 @@ def forward(
             attention_mask=attention_mask,
             **cross_attention_kwargs,
         )
-        if self.use_ada_layer_norm_zero:
+        if self.num_embeds_ada_norm is not None and self.norm_type == "ada_norm_zero":
```

Comment:
I think it is guaranteed  `self.num_embeds_ada_norm` is not `None` here - it will get a value error during `__init__`
```suggestion
        if self.norm_type == "ada_norm_zero":
```